snuba/request_schema.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
```

Comment:
This entire file is just a cut and paste of what was in schemas.py without functional changes.
This is to break the circular dependency between schemas.py and query/extensions.py:
QueryExtension depends on Schema (because it returns extension schemas)
RequestSchema depends on Schema (obviously)
RequestSchema.build_with_extension (factory method) depends on QueryExtension (input parameter).

Thus, as long as RequestSchema and Schema are in the same file there is a circular dependency that does not need to exist since both RequestSchema and QueryExtension are higher level concepts than Schema.

Reply:
Makes sense.